### PR TITLE
added new policy changes in 1903

### PR DESCRIPTION
**Pull request by RAJU2529**
added the group policy settings  changes from 1809 to  may 1903 ,
under system and windows components

### CODE REVIEW

**Comment by RAJU2529** [comment]
@illfated @jvsam, i compared the administrative templates of 1809 and may 1903 thoroughly and then i added  new policy changes  from 1809 to may 1903

File: windows/client-management/new-policies-for-windows-10.md [hunk 1]
```diff
@@ -25,6 +25,33 @@ ms.topic: reference
 
 Windows 10 includes the following new policies for management. [Download the complete set of Administrative Template (.admx) files for Windows 10](https://www.microsoft.com/download/100591).
 
+## New Group Policy settings in Windows 10, version 1903
+
+The following Group Policy settings were added in Windows 10, version 1903:
+
+**System**
+
+- System\Service Control Manager Access\Security Securty\Enable svchost.exe migitation options
```
Comment (illfated) [inline windows/client-management/new-policies-for-windows-10.md]:
```suggestion
- System\Service Control Manager Settings\Security Settings\Enable svchost.exe mitigation options
```
typo corrections

Comment (RAJU2529) [reply windows/client-management/new-policies-for-windows-10.md]: accepted, thanks for finding mistake

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing and style checks needed for this PR. Thanks.

**Comment by RAJU2529** [comment]
@e0i  When the new content will be published

**Comment by illfated** [comment]
```MarkDown
---
title: New policies for Windows 10 (Windows 10)
description: Windows 10 includes the following new policies for management.
ms.assetid: 1F24ABD8-A57A-45EA-BA54-2DA2238C573D
ms.reviewer: 
manager: dansimp
ms.author: dansimp
keywords: ["MDM", "Group Policy"]
ms.prod: w10
ms.mktglfcycl: manage
ms.sitesec: library
author: dansimp
ms.localizationpriority: medium
ms.date: 10/24/2017
ms.topic: reference
---
```
CC: @Dansimp

**Comment by illfated** [comment]
#sign-off

**Comment by e0i** [comment]
cc @Dansimp

**Comment by martyav** [comment]
Hi, @e0i. I'm on Dan's team and was recently asked to help clean up the PR backlog.

**Pull request merged**